### DISCUSSION TITLE
Forgot the "Back to model" links on image post submission, editing images, editing model, etc.

### DISCUSSION THREAD

**Discussion by WASasquatch** [civitai/civitai#475] [Ideas]
See title. You gotta go to your models, or use back, etc, to return to your model. Only submitting changes takes you back to the model.

**Comment by Nenotriple** [comment]
Perhaps it could be labeled differently, but the X (exit) button in the top right seems to do what you're suggesting.

**Reply by WASasquatch** [reply]
Oh, I didn't even notice it at all! Think my resolution is too big.
